packages/gel/src/baseConn.ts

```diff
@@ -482,6 +482,19 @@ export class BaseRawConnection {
     throw new errors.ProtocolError("invalid input codec");
   }
 
+  private isInTransaction(): boolean {
```

Comment:
nit: just for the consistency with the surrounding code I'd name this `_isInTransaction`